tests/codeception/acceptance/Paragraphs/MissionStatementCest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Tests on the mission statement paragraph type.
+ */
+class MissionStatementCest {
+
+  /**
+   * The paragraph should display on a basic page.
+   */
+  public function testMissionStatement(AcceptanceTester $I) {
+    $I->logInWithRole('site_manager');
+    $paragraph = $I->createEntity([
+      'type' => 'cardinal_mission_statement',
+      'su_mission_text' => 'This is the mission statement whether you choose to accept it or not.',
+      'su_mission_cta' => [
+        'uri' => 'http://google.com',
+        'title' => 'Verify your identity',
+      ],
+      'su_mission_lower_text' => 'This paragraph will self destruct at the end of this test',
+    ], 'paragraph');
+
+    $node = $I->createEntity([
+      'type' => 'stanford_page',
+      'title' => 'Bond, Drupal Bond',
+      'su_page_components' => [
+        'target_id' => $paragraph->id(),
+        'entity' => $paragraph,
+        'settings' => json_encode([
+          'row' => 0,
+          'index' => 0,
+          'width' => 12,
+          'admin_title' => 'Card',
+        ]),
+      ],
+    ]);
+
+    $I->amOnPage($node->toUrl()->toString());
+    $I->canSee('This is the mission statement whether you choose to accept it or not');
+    $I->canSeeLink('Verify your identity');
+    $I->canSee('This paragraph will self destruct at the end of this test');
+  }
```

Comment:
Should there be a test for the C2A URI?

Reply:
sounds good to me :+1:

Reply:
https://github.com/SU-HKKU/cardinal_service_profile/pull/23/commits/5fb1c99bf658a441cfc913bd3ec59f74a91e9a2b#diff-649603cd48bdfc49ea5d40f7b4e2482bR42